web/src/interfaces.ts

```diff
@@ -64,3 +64,22 @@ export interface Customer {
   address: string;
   numOngoingCommits: number;
 }
+
+/**
+ * MerchantPayload Interface that contains the fields of the payload that
+ * would be sent to the server to add the merchant to the database.
+ */
+export interface MerchantPayload {
+  name: string;
+  email: string;
+  vpa: string;
+}
+
+/**
+ * MerchantResponse Interface that contains the fields of the Response that
+ * client side would receive.
+ */
+export interface MerchantResponse extends MerchantPayload {
```

Comment:
Probably the name "addMerchantResponse" is more appropriate to how you attempt to use it?

Reply:
This response is supposed to be a representation for a Merchant object that is used in the HTTP response. So I think `MerchantResponse` would be more accurate.